orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/AmazonImageFinder.java

```diff
@@ -131,6 +133,7 @@ public String getCloudProvider() {
 
   static class StageData {
     @JsonProperty Collection<String> regions;
+    public String account;
```

Comment:
Is it correct/required to add `@JsonProperty` here?

Reply:
it isn't necessary (your test case proves that)

Reply:
Yeah, was curious whether there's some gap so that even with a passing test, it could break in the 'real world.'